### DISCUSSION TITLE
Consortium setup

### DISCUSSION THREAD

**Discussion by joefeewln** [MCLD/greatreadingadventure#1002] [Help!]
Hello, our consortium is looking to set this up for our member libraries.  Is GRA granular enough to allow each library to have their own programs and settings that other libraries can not change? Or, will we need to install a GRA instance for each library?

**Comment by k7hpn** [comment]
Hello, @joefeewin!

Currently we don't really have the multi-tenancy that you're talking about. Our situation is similar (we run our summer reading program for all library systems in Maricopa County) but we just allow every system to make their own content in the software and encourage participants to engage with any libraries in Phoenix - regardless of system. We track who makes changes to what so if we had one library changing another library's items we'd be able to contact the admin who made the changes and ask them to stop.

For now, individual instances for each library is probably the approach if you need that kind of full separation. This might be made easier deploying via Docker (that's how we deploy here).
